docs/enterprise-setup/implementation-guide.md

```diff
@@ -81,11 +81,162 @@ To configure basic auth (deploy without SSO), remove the entire `auth:` section
 
 </details>
 
+#### Configuring the Airbyte Database
+
+For Self-Managed Enterprise deployments, we advise against using the default Postgres database (`airbyte/db`) that Airbyte spins up within the Kubernetes cluster. For production, you should instead use a dedicated instance for better reliability, and backups (such as AWS RDS or GCP Cloud SQL).
+
+We assume in the following that you've already configured a Postgres instance:
+
+1. In the `charts/airbyte/values.yaml` file, disable the default Postgres database (`airbyte/db`):
+
+```yaml
+postgresql:
+    enabled: false
+```
+
+2. In the `charts/airbyte/values.yaml` file, enable and configure the external Postgres database:
+
+```yaml
+externalDatabase:   
+    host: ## Database host
+    user: ## Non-root username for the Airbyte database
+    database: db-airbyte ## Database name
+    port: 5432 ## Database port number 
+```
+
+For the non-root user's password which has database access, you may use `password`, `existingSecret` or `jdbcUrl`. We recommend using `existingSecret`, or injecting sensitive fields from your own external secret store. Each of these parameters is mutually exclusive: 
```

Comment:
> injecting sensitive fields

Are we going to document (could be in the future) how we recommend doing this?

Reply:
We have to :)